src/app/pipes/alm-user-name.pipe.ts

```diff
@@ -0,0 +1,13 @@
+import { User } from './../models/user';
+import { Pipe, PipeTransform } from '@angular/core';
+
+// tslint:disable-next-line:use-pipe-transform-interface
+@Pipe({ name: 'almUserName', pure: true })
+export class AlmUserName implements PipeTransform {
+  transform(userObj: User | null = null, notFound: string = 'User not found'): string {
+    if (typeof(userObj) === 'undefined' || userObj === null) {
+      return notFound;
+    }
+    return userObj.attributes.fullName;
```

Comment:
If user object is there then fullName has to be there. If it's null then the output should be empty string. I haven't handle that though. Updating.